service-base/src/main/java/org/eclipse/hono/service/http/AbstractHttpEndpoint.java

```diff
@@ -56,6 +57,10 @@
      * The name of the URI path parameter for the device ID.
      */
     protected static final String PARAM_DEVICE_ID = "device_id";
+    /**
+     * The key that is used to put the if-Match ETags values to the RoutingContext.
+     */
+    protected static final String KEY_ETAGS_VERSION = "KEY_ETAGS_VERSION";
```

Comment:
We might want to name this a bit more neutral, maybe "resource-version".

Reply:
Note that it's only used internally in  the _HTTPEndpoint_ to store the values in the `RoutingContext`. I thought it made sense to match the header name, but both works :+1:

Reply:
I don't have strong opinion here, both works for me.

Reply:
FMPOV this should be named `KEY_RESOURCE_VERSION` as that is what we store under this key. The fact that it is determined from the If-Match header is an implementation details ...

Reply:
updated to the suggested value